src/flint/fmpz.jl

```diff
@@ -1249,10 +1248,17 @@ function rising_factorial(x::fmpz, y::Int)
     y < 0 && throw(DomainError("Argument must be non-negative: $y"))
     z = fmpz()
     ccall((:fmpz_rfac_ui, :libflint), Nothing,
-          (Ref{fmpz}, Ref{fmpz}, UInt), z, x, y)
+          (Ref{fmpz}, Ref{fmpz}, UInt), z, x, UInt(y))
     return z
 end
 
+@doc Markdown.doc"""
+    rising_factorial(x::fmpz, y::fmpz)
+> Return the rising factorial of $x$, i.e. $x(x + 1)(x + 2)\ldots (x + n - 1)$.
```

Comment:
Isn't `\cdots` more curstomary here? Also, mismatch between `y` and `n`.
Oh, I see you copied the docstring from below... I know this is done in other places in this project, but I think we shouldn't duplicate docstrings like that. But I will open a separate issue to discuss this, which is independant of your PR.

Reply:
I was going to leave the extra docstrings out, but thought maybe someone would complain I didn't put them in.

I'll change it to cdots.